plugins/autowarmth.koplugin/main.lua

```diff
@@ -43,11 +43,13 @@ local function frac(x)
     return x - math.floor(x)
 end
 
+-- Local variable that shall survive reloading of the plugin but not a restart
+local fl_turned_off = nil -- true/false if someone (AutoWarmth, gesture ...) has toggled the frontlight
```

Comment:
You can leave it in AutoWarmth, as long as you always access it as `AutoWarmth.fl_turned_off`, and never `self.fl_turned_off`.

Reply:
I knew I was missing something.